spec/cancan/controller_resource_spec.rb

```diff
@@ -501,10 +501,10 @@ class Model < ::Model; end
 
     it 'allows full find method to be passed into find_by option' do
       model = Model.new
-      allow(Model).to receive(:find_by_name).with('foo') { model }
+      allow(Model).to receive(:find_by_name!).with('foo') { model }
 
       params.merge!(action: 'show', id: 'foo')
-      resource = CanCan::ControllerResource.new(controller, find_by: :find_by_name)
+      resource = CanCan::ControllerResource.new(controller, find_by: :name)
       resource.load_resource
       expect(controller.instance_variable_get(:@model)).to eq(model)
```

Comment:
This test case was not worked before. It was same as https://github.com/CanCanCommunity/cancancan/pull/461/files#diff-eba4e6277d1f300a2baa8a35cef7ef0fR492.

so I updated test case.

Reply:
this is actually a bit misleading. could you restore the intention of the previous test? but maybe use `allow(Model).to receive(:search).with('foo') { model }` on line 504 and on line 507 you use `find_by: :search` which should reflect the previous behaviour.

Reply:
Ah I see. I didn't notice that. Got it.